components/core/tools/scripts/lib_install/check-cmake.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Get the installed cmake version string
+cmake_version=$(cmake --version | head -n1 | awk '{print $3}')
+cmake_major_version=$(echo "$cmake_version" | cut -d. -f1)
```

Comment:
Would it be better to write
```suggestion
cmake_major_version=$(cmake -E capabilities | jq ".version.major")
```